app/controllers/provider_interface/provider_users_invitations_controller.rb

```diff
@@ -4,7 +4,7 @@ class ProviderUsersInvitationsController < ProviderInterfaceController
     before_action :require_manage_user_permission!
 
     def edit_details
-      @wizard = wizard_for(current_step: 'details')
+      @wizard = wizard_for(current_step: 'details', single_provider: current_provider_user.providers.size == 1)
```

Comment:
This needs to be `current_provider_user.authorisation.providers_that_actor_can_manage_users_for` or we get the checkbox when the user has 2 providers but can only manage for 1

Reply:
@duncanjbrown thanks! 
Done here https://github.com/DFE-Digital/apply-for-teacher-training/pull/2606/commits/10aa54773817413adb9aecdba831e4e2c45b912d